docs/source/install/deb.rst

```diff
@@ -4,39 +4,12 @@ Ubuntu / Debian
 This guide provides step-by step instructions for installing |st2| on a single Ubuntu/Debian 64 bit system per
 the :doc:`Reference deployment </install/overview>`.
 
-.. rubric:: TL;DR
-
-That's OK! You're busy, we get it. How do you just get started? Get yourself a clean box, and run this command:
-
-::
-
-   curl -sSL https://stackstorm.com/packages/install.sh | bash -s -- --user=st2admin --password=<CHANGEME>
+.. note:: `Use the Source, Luke! <http://c2.com/cgi/wiki?UseTheSourceLuke>`_ We strive to keep the documentation current, yet the best way to find out what really happens
```

Comment:
Am wondering about a wording change here. Could maybe be "...current, but the best way to find out exactly what our recommended installation does is to look at the code..."

That's not quite right either, but the point is, that this page is now separate from the TL;DR section. The line as it is makes more sense if it was still following th old structure. 

Meh. But don't worry too much. Change is good.